src/Jackett.Common/Definitions/pttime.yml

```diff
@@ -0,0 +1,148 @@
+    - {id: 407, cat: TV/Sport, desc: "Music/音乐"}
+    - {id: 408, cat: Audio, desc: "Edu/教育"}
+    - {id: 409, cat: Other, desc: "Resource/资源"}
+    - {id: 410, cat: XXX, desc: "9kg/生理"}
+    - {id: 411, cat: Books, desc: "Other/其它"}
+
+  modes:
+    search: [q]
+    tv-search: [q, season, ep, imdbid]
+    movie-search: [q, imdbid]
+    music-search: [q]
+    book-search: [q]
+
+settings:
+  - name: username
+    type: text
+    label: Username
+  - name: password
+    type: password
+    label: Password
+  - name: freeleech
+    type: checkbox
+    label: Search freeleech only
+    default: false
+  - name: info_tpp
+    type: info
+    label: Results Per Page
+    default: For best results, change the <b>Torrents per page:</b> setting to <b>100</b> on your account profile.
+
+login:
+  path: takelogin.php
+  method: post
+  inputs:
+    loginmethod: username
+    username: "{{ .Config.username }}"
+    password: "{{ .Config.password }}"
+    dutime: month
+    ssl: yes
+  error:
+    - selector: td.embedded:has(h2:contains("失败"))
+    - selector: td.embedded:has(h2:contains("Failed"))
+  test:
+    path: index.php
+    selector: a[href="logout.php"]
+
+search:
+  paths:
+    - path: torrents.php
+  inputs:
+    $raw: "{{ range .Categories }}cat{{.}}=1&{{end}}"
+    search: "{{ if .Query.IMDBID }}{{ .Query.IMDBID }}{{ else }}{{ .Keywords }}{{ end }}"
+    # 0 incldead, 1 active, 2 dead
+    incldead: 0
+    # 0 all, 1 normal, 2 free, 3 2x, 4 2xfree, 5 50%, 6 2x50%, 7 30%
+    spstate: "{{ if .Config.freeleech }}2{{ else }}0{{ end }}"
+    # 0 title, 1 descr, 3 uploader, 4 imdburl (searching imdburl does not work with tt1234567, but descr is good)
+    search_area: "{{ if .Query.IMDBID }}1{{ else }}0{{ end }}"
+    # 0 AND, 1 OR, 2 exact
+    search_mode: 0
+
+  rows:
+    selector: table.torrents > tbody > tr:has(table.torrentname)
+
+  fields:
+    category:
+      selector: a[href^="?cat="]
+      attribute: href
+      filters:
+        - name: querystring
+          args: cat
+    title:
+      selector: a[href^="details.php?id="]
+    title:
+      optional: true
+      selector: a[title][href^="details.php?id="]
+      attribute: title
+    details:
+      selector: a[href^="details.php?id="]
+      attribute: href
+    download:
+      selector: a[href^="download.php?id="]
+      attribute: href
+    imdb:
+      selector: div.imdb_100 > a
+      attribute: href
+    date:
+      # time type: time elapsed (default)
+      selector: td:nth-child(5) > span[title]
+      attribute: title
+      optional: true
+      filters:
+        - name: append
+          args: " +08:00" # CST
+        - name: dateparse
+          args: "2006-01-02 15:04:05 -07:00"
+    date:
+      # time added
+      selector: td:nth-child(5):not(:has(span))
+      optional: true
+      filters:
+        - name: append
+          args: " +08:00" # CST
+        - name: dateparse
+          args: "2006-01-0215:04:05 -07:00"
```

Comment:
Is there a space missing between the date and time?
"2006-01-0215:04:05 -07:00" vs
"2006-01-02 15:04:05 -07:00"

Reply:
>Is there a space missing between the date and time?

no, when the torrentlist is in time-added mode,  there is usually a `<br>` in-between, which dateparse ignores, so this ok.